dcrpm/util.py

```diff
@@ -172,13 +173,10 @@ def run_with_timeout(cmd, timeout, raise_on_nonzero=True, raise_on_timeout=True)
     `raise_on_timeout` is true, raises a DcRPMException if `cmd` times out.
     """
     _logger.debug("Running %s", cmd)
-    cmdname = cmd.split()[0]
+    argv = shlex.split(cmd)
```

Comment:
This seems like a bit of a hack. "Try to parse the way the shell parses" instead of just updating all the call sites... My instinct would be to update all the call sites instead. This is certainly somewhat better than we have today, but given the low-risk (we take no user-input, we're not running SUID, etc.), I don't feel like we have a strong need for a stop-gap right away, and would rather update our callers to pass lists around... That said if people feel strongly, I won't block it... it just doesn't seem like the right approach to me.

Reply:
I concur with @jaymzh - transforming cmd from a single string to a list of strings isn't much of a stretch here, and it can eliminate the format string expansion in most places. @markdrayton : could you do that?

Reply:
Yep, I have an update almost ready. I got caught up on some other stuff but will try to update the PR today or tomorrow.